src/platform/cannonlake/include/platform/memory.h

```diff
@@ -245,10 +245,30 @@
 #define HEAP_HP_BUFFER_COUNT \
 			(HEAP_HP_BUFFER_SIZE / HEAP_HP_BUFFER_BLOCK_SIZE)
 
+#if defined(CONFIG_DMIC)
+#define SOF_TEXT_DMIC_SIZE 0x2000
```

Comment:
Do these sizes need to be multiple of 0x1000? If they need to be, it's better better to mention such requirement in comment. If it could be e.g. multiple of 0x100 should we try find out for what's worst-case for gcc or xt-xcc compile and use those numbers?

Reply:
@singalsu yes, 0x1000 needed. @mmaka1  ok, I noted it for another PR.